Source/SpectrumAnalysis/KTChannelAggregator.cc

```diff
@@ -0,0 +1,155 @@
+/*
+ * KTChannelAggregator.cc
+ *
+ *  Created on: Jan 25, 2019
+ *      Author: P. T. Surukuchi
+ */
+
+#include "KTChannelAggregator.hh"
+
+#include "KTLogger.hh"
+
+namespace Katydid
+{
+    KTLOGGER(agglog, "KTChannelAggregator");
+
+    // Register the processor
+    KT_REGISTER_PROCESSOR(KTChannelAggregator, "channel-aggregator");
+
+    KTChannelAggregator::KTChannelAggregator(const std::string& name) :
+            KTProcessor(name),
+            fPhaseChFrequencySumSlot("fft", this, &KTChannelAggregator::SumChannelVoltageWithPhase, &fSummedFrequencyData),
```

Comment:
The order of the initialization list needs to match the order in which member variables are defined.  Otherwise the compiler complains.

Reply:
Fixed in 8fea59e0bfb1ea3ac6f519d00ff17095c12cdddb.